src/main/java/de/guntram/mcmod/durabilityviewer/itemindicator/TREnergyIndicator.java

```diff
@@ -19,8 +20,8 @@ public class TREnergyIndicator implements ItemIndicator {
 
     public TREnergyIndicator(ItemStack stack) {
         this.stack = stack;
-        if (stack.getItem() instanceof EnergyHolder) {
-            maxEnergy = ((EnergyHolder) stack.getItem()).getMaxStoredPower();
+        if (stack.getItem() instanceof EnergyStorage) {
+            maxEnergy = ((EnergyStorage) stack.getItem()).getCapacity();
```

Comment:
Maybe 

```suggestion
        if (stack.getItem() instanceof EnergyStorage energyStorage) {
            maxEnergy = energyStorage.getCapacity();
```

(not sure which java version this mod uses)